draft-ietf-masque-h3-datagram.md

```diff
@@ -260,7 +260,8 @@ This specification defines the "data stream" of an HTTP request as the
 bidirectional stream of bytes that follow the headers in both directions. In
 HTTP/1.x, the data stream consists of all bytes on the connection that follow
 the blank line that concludes either the request header section, or the 2xx
-(Successful) response header section. In HTTP/2 and HTTP/3, the data stream of
+(Successful) response header section. (Note that HTTP pipelining is not
```

Comment:
well it's not just pipelining, there's no means to send more than 1 request because the connection ceases to be HTTP after the request or response

Reply:
I was under the impression that "send more than 1 request" and "pipelining" were the exact same thing. Am I confused?

Reply:
I think the distinction between pipeling and non pipelining for a persistent connection is that you can send the additional HTTP request(s) after each other with pipelining without receiving any or the full response of the previous request. https://www.ietf.org/archive/id/draft-ietf-httpbis-messaging-19.html#name-pipelining

So, the restriction is really that it is not possible to subsequent HTTP requests after one that enables HTTP Datagram.

Reply:
Thanks, I've rewritten the text to replace pipelining with multiple requests

Reply:
I feel like there should be some language in HTTP core that bans pipelining with `Connection: close` and `Connection: upgrade` that we would hopefully be covered by.

Reply:
@vasilvv I wasn't able to find that text, could you point me at it please?